grpcbp/client_middlewares.go

```diff
@@ -0,0 +1,65 @@
+package grpcbp
+
+import (
+	"context"
+	"strings"
+
+	"github.com/opentracing/opentracing-go"
+	"github.com/reddit/baseplate.go/ecinterface"
+	"github.com/reddit/baseplate.go/tracing"
+	"google.golang.org/grpc"
+)
+
+type MonitorClientArgs struct {
+	ServiceSlug string
+}
+
+func MonitorInterceptor(args MonitorClientArgs) grpc.UnaryClientInterceptor {
```

Comment:
MonitorInterceptorArgs?

Reply:
I think "Monitor" is an incorrect name for this.  This does _tracing_ which is not monitoring.  Monitor, to me at least, implies metrics.

TracingInterceptorUnary?
Or, even better, TraceUnary?

Reply:
I initially named it `TracingInterceptor`. Tracing is the better name but we use `Monitor[ed]` across the baseplate repository. Maybe we can keep it consistent and rename them all or is there a more subtle difference with other monitor middlewares?
I also okay calling this tracing right away since this is a new package.

I assume we would call it Unary to allow for a Stream implementation too?